### DISCUSSION TITLE
API 2.0 for GTR-3 Pro?

### DISCUSSION THREAD

**Discussion by Mostalk** [zepp-health/zeppos-forum#42] [Q&A] [answered]
Hello, should we expect Zepp OS 2.0 API for GTR-3 PRO?

**Comment by MyoungXUE** [comment, marked as answer]
Hello, thank you for your support to the forum.

Due to the hardware limitation of GT3 series, it is unable to upgrade to ZeppOS 2.0

**Reply by Mostalk** [reply]
No updates for GT3 anymore in future?

**Reply by MyoungXUE** [reply]
We will continue maintaining new fixes for GT3.

The reason is that we are a tiny company compared to those tech giants, and we have no resources to keep backward upgrading. This is also restricted due to the existing hardware design for each different device.

You can join our eventbrite session and we will tell you more about it. https://www.eventbrite.com/e/zepp-os-30-apis-product-roadmap-technical-updates-tickets-510416398187

**Reply by Mostalk** [reply]
Good. Thanks for the invitation. I'll try to come and hear about the future of Zepp OS. By just buying a $200 watch (price at my local store for a GTR-3 Pro) I was hoping for longer software support with an OS update :)

Eventbrite says Zepp OS 3.0. Will GT4 support this OS or will you have to buy GT5 then? :D

**Reply by MyoungXUE** [reply]
Sorry, we are not aware of any specific plans for the GT4 upgrade at this time either. 😅

**Comment by htoooth** [comment]
hi @Mostalk, 
you can try the [zapp-framework](https://github.com/j-piasecki/zapp-framework).

**Reply by Mostalk** [reply]
Hi, thanks for it. An interesting project.
